mkShapesRDF/shapeAnalysis/latinos/PlotFactory.py

```diff
@@ -672,7 +665,10 @@ def makePlot(
                                 if "/" in values:
                                     variations = map(float, values.split("/"))
                                 else:
-                                    variations = (float(values), 2.0 - float(values))
+                                    if float(values) < 1e-8:
+                                        print("lnN cannot be zero! Check nuisance", nuisacen, file=sys.stderr)
```

Comment:
I guess it's a typo: `nuisacen` -> `nuisance`

Reply:
Thank for spotting that!